### PR TITLE
feat: Add support for closing in-context sidebar [BD-38] 

### PR DESCRIPTION
**Pull request by xitij2000**
Use postmessage-based API to send message to close learning MFE sidebar.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @xitij2000! I've created [BLENDED-1039](https://openedx.atlassian.net/browse/BLENDED-1039) to keep track of it in Jira.   More details are on the [BD-38](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2790490201/BD-38+Discussions+In-Context+Discussion+Moderation+Phase+2) project page.


When this pull request is ready, tag your edX technical lead.

**Comment by Cup0fCoffee** [comment]
:+1:

- [x] I tested this
- [x] I read through the code

File: src/discussions/common/ActionsDropdown.test.jsx [hunk 1]
```diff
@@ -39,18 +50,16 @@ const canPerformActionTestData = ACTIONS_LIST
           buildParams[conditionKey] = conditionValue;
         });
     }
-    return buildTestContent(buildParams)
-      .map(commentOrPost => ([defaultMessage, commentOrPost, action]));
+    return buildTestContent(buildParams, { label: defaultMessage, action });
   })
   .flat();
-
+// console.log(canPerformActionTestData);
```
Comment (mehaknasir) [inline src/discussions/common/ActionsDropdown.test.jsx:56]: can we remove this line?

**Review by mehaknasir — APPROVED** [review]
lgtm just a nit.

File: src/discussions/common/ActionsDropdown.test.jsx (hunk 1 above)
Comment (xitij2000) [reply src/discussions/common/ActionsDropdown.test.jsx:56]: Yeah, sorry I think I left in some log statements. Will remove.

**Comment by edx-webhook** [comment]
@xitij2000 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-discussions&entry.867055334=https://github.com/openedx/frontend-app-discussions/pull/40&entry.1484655318=https://github.com/xitij2000&entry.752974735=2021-12-07+13:13&entry.1917517419=2022-04-01+08:11&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**